libs/dao/src/main/java/com/akto/dto/CollectionConditions/CollectionCondition.java

```diff
@@ -19,7 +28,7 @@ public CollectionCondition(Type type, Operator operator) {
     public abstract Map<CollectionType, Bson> returnFiltersMap();
 
     public enum Type {
-        API_LIST
+        API_LIST, METHOD, PARAM, TIMESTAMP
     }
```

Comment:
How are APIs that are deleted or merged handled here?

Reply:
we have a daily CRON which which updates all the collections based on these conditions. Any changes in STI/api_info/sample_data etc. due to deletion/merging occur will be reflected so.